src/blue.ts

```diff
@@ -132,6 +132,15 @@ export function blueProxyFactory(broker: MembraneBroker, hooks: typeof Reflect)
 
         constructor(target: BlueProxyTarget) {
             this.target = target;
+        }
+        initialize(shadowTarget: BlueShadowTarget) {
```

Comment:
the `initialize` method used by red handler is now also implemented in blue handler, although it does something slightly different. It is simply correcting the value of the function's prototype descriptor to point to the right value. This is needed because by default, when declaring a function, it's `.prototype` data property is non-configurable, which messes up with the logic that we have implemented in `getOwnPropertyDescriptor` trap to preserve the language object semantics for non-configurable properties.